Makefile

```diff
@@ -147,6 +147,14 @@ projection_ho: projection.pdf
 	mkdir -p $(HANDOVER_DIR)
 	cp -f $< $(HANDOVER_FN)
 
+
+.PHONY: tex
+tex: 
+	sudo apt install chktex
+
+.PHONY: lint
+lint: tex
```

Comment:
You only install `chktex`, but you do not use `chktex` to do something. For example, you may add lines like `chetex projection.tex` to ask for `chktext` to check `projection.tex` for you.

After completing this, you need to edit `.github/workflows/build_doc.yml` to append lines like `make lint` to make the lint checking automatic via Github Action.